pkg/http_endpoint/v1/router.go

```diff
@@ -56,26 +60,645 @@ func authRouter(clients *common.ClientContainer,
+                                "Unauthorized", err.Error())
+                        return
+                // If any other error happened -> return 500 error
+                default:
+                        log.Logger.Error(err)
+                        common.WriteErrorToResponse(w, http.StatusInternalServerError,
+                                http.StatusText(http.StatusInternalServerError),
+                                "Internal server error occured")
+                        return  
+                }       
+        }
+	organizationID := chi.URLParam(r, "organizationID")
+	ID, _ := strconv.Atoi(organizationID)
+	err = clients.Grafana.DeleteOrgUser(ID)
+	if err != nil {
+		log.Logger.Error(err)
+		common.WriteErrorToResponse(w, http.StatusInternalServerError,
+			http.StatusText(http.StatusInternalServerError),
+			"Internal server error occured")
+		return
+	}
+	w.WriteHeader(http.StatusOK)
+	w.Write([]byte(fmt.Sprintf("Sucessfully deleted user in organization by ID: %v", organizationID)))
+		}
+}
+
+//GetOrganizationUser gets the user by organization
+func GetOrganizationUser(clients *common.ClientContainer) http.HandlerFunc {
+                return func(w http.ResponseWriter, r *http.Request) {
+	err := clients.Grafana.DoLogon()
+        if err != nil {
+                switch err.(type) {
+                // common.InvalidUsernamePassword means, that user provided invalid
+                // username/password. We have to return 401 error then
+                case common.InvalidUsernamePassword:
+                        common.WriteErrorToResponse(w, http.StatusUnauthorized,
+                                "Unauthorized", err.Error())
+                        return  
+                // If any other error happened -> return 500 error
+                default:
+                        log.Logger.Error(err)
+                        common.WriteErrorToResponse(w, http.StatusInternalServerError,
+                                http.StatusText(http.StatusInternalServerError),
+                                "Internal server error occured")
+                        return  
+                }       
+        }
+	organizationID := chi.URLParam(r, "organizationID")
+	ID, _ := strconv.Atoi(organizationID)
+	orglist, err := clients.Grafana.GetOrgUsers(ID)
+	if err != nil {
+		log.Logger.Error(err)
+		common.WriteErrorToResponse(w, http.StatusInternalServerError,
+			http.StatusText(http.StatusInternalServerError),
+			"Internal server error occured")
+		return
+	}
+	orgs, _ := json.Marshal(orglist)
+	w.WriteHeader(http.StatusOK)
+	w.Write(orgs)
+		}
+}
+
+func adminRouter(clients *common.ClientContainer,
+		authMiddleware func(http.Handler) http.Handler, secret string) *chi.Mux {
+	r := chi.NewRouter()
+	r.Use(authMiddleware)
+	r.Use(httpAuth.AdminAuthenticationMiddleware(secret))
+
+	// routes for datasourcess
+	r.Route("/datasources", func(r chi.Router) {
```

Comment:
We don't have this route in admin section http://petstore.swagger.io/?url=https://raw.githubusercontent.com/zaletniy/visualization-api/master/doc/visualization-api-1.0.0.yaml
it has to be moved to separate router